### PR TITLE
Integrate fixed apr staking contract

### LINKED ISSUE #274: GameFi fixed APR page
User stories:

Stake
- As a user I want to connect my wallet so that I can stake in the vaults.
- As a user I want to see the available vaults so that I can select where I will stake my tokens.
- As a user I want to be see a button that will redirect to an exchange incase I don’t have enough tokens so that I can buy easily.
- As a user I want to see accurate details about the pool so that I can wisely decide which pool I will stake.
- As a user I want to see different staking durations so that I can choose how long my tokens will be lock.
- As a user I want to see an input field button so that I can input my desired amount to stake.
- As a user I want to see data such as APR/APY, unstaking fee, staked tokens and balance so that I will know how much I will earn from the vault.
- As a user I want to see stake button that opens metamask so that I can confirm
- As a user I want to see success or cancel modal/toast so that I can confirm the status of my transaction
- As a user I want to see a logout button so that I can disconnect my wallet.

Withdraw
- As a user I want to see a dashboard so that I can see my accumulated rewards
- As a user I want to see the remaining days of my staking so that I am aware when I can withdraw my tokens.
- As a user I want to see a withdraw button where I can click to withdraw my stake rewards.

TO DO::
Staking
- [x] show vaults
- [x] show fetched data in GameFi pages
- [x] show apr/duration options
- [x] apply estimated computation rates based on APR/duration to select by user
- [x] link to exchange on staking token
- [x] staking modal prompt

Withdraw
- [x] list user stakes as cards
- [x] show day duration, day staked, & days remaining on cards
- [x] withdraw modal prompt

### PR DESCRIPTION
**Pull request by medardm**
closes #274  #268

### CODE REVIEW

**Review by EAdornado — APPROVED** [review]

**Review by EAdornado — APPROVED** [review]
All goods!

File: src/state/pools/index.ts [hunk 1]
```diff
@@ -105,52 +111,65 @@ export const fetchPoolsStakingLimitsAsync = () => async (dispatch, getState) =>
...
     dispatch(setPoolsUserData(userData))
   }
 
 export const updateUserAllowance =
-  (sousId: number, account: string): AppThunk =>
+  (sousId: number, account: string, chain: string): AppThunk =>
   async (dispatch) => {
-    const allowances = await fetchPoolsAllowance(account)
+    const allowances = await fetchPoolsAllowance(account, chain)
+    console.log('updateUserAllowance: ', allowances)
```
Comment (swengr-janan) [inline src/state/pools/index.ts]: kuys pa remove console.log

**Review by swengr-janan — CHANGES_REQUESTED** [review]

**Review by EAdornado — APPROVED** [review]
All goods!

**Review by kojiadrianojr — APPROVED** [review]

**Review by swengr-janan — CHANGES_REQUESTED** [review]
Typo
Screen Shot 2022-07-20 at 4 35 30 PM

**Review by swengr-janan — APPROVED** [review]
AGITH!

**Review by EAdornado — APPROVED** [review]
All Gooooods!

**Pull request merged**